specification/cognitiveservices/OpenAI.Inference/models/embeddings.create.cadl

```diff
@@ -24,18 +24,17 @@ model EmbeddingsOptions {
     as we have observed inferior results when newlines are present.
     """)
     input: string | string[];
+
+    @added(ServiceApiVersions.v2022_12_01)
```

Comment:
This is untrue, as this field is already available in the preview API version and was not added in 2022-12-01. I would remove the line